config/config_python.py

```diff
@@ -0,0 +1,34 @@
+import pip
+import pkg_resources
+
+def main():
+    # Install gslab_tools if it's not installed yet or it has the wrong version
+    if not check_gslab_tools('4.1.0'):
+        pip.main(['install', 'git+http://git@github.com/gslab-econ/gslab_python.git@4.1.0'])
+
+    # Install required packages using pip
+    packages = {'pyyaml': True}
+    installed_packages = [pkg.key for pkg in pip.get_installed_distributions()]
+    for pkg in packages.keys():
+        if pkg not in installed_packages or packages[pkg]:
+            pip.main(['install', '--upgrade', pkg])
+
+def check_gslab_tools(gslab_python_version):
```

Comment:
This makes a pretty strong promise about backwards compatibility. Maybe we just force installation of the current version?

Reply:
Do you mean `gslab_tools` or the other packages?

Reply:
I guess both, but I'm mostly concerned about `gslab_tools`.

Reply:
So we do not check the revision any more, but force the users to install what's currently in `master`?

Reply:
Yes. Except we should force installation of the v4.1.0 release.

```python
pip.main(['install', '--upgrade', 
          'git+http://git@github.com/gslab-econ/gslab_python.git@v4.1.0'])
```